prophet_bento_timeseries/bentoml-kubeflow-prophet.ipynb

```diff
@@ -0,0 +1,1638 @@
+      "  Downloading docutils-0.15.2-py3-none-any.whl (547 kB)\n",
+      "Collecting ply<4.0,>=3.4\n",
+      "  Downloading ply-3.11-py2.py3-none-any.whl (49 kB)\n",
+      "Collecting MarkupSafe>=0.9.2\n",
+      "  Downloading MarkupSafe-1.1.1-cp37-cp37m-manylinux1_x86_64.whl (27 kB)\n",
+      "Building wheels for collected packages: fbprophet, psutil, cerberus, alembic, python-json-logger, sqlalchemy-utils, holidays, thriftpy2, pymeeus\n",
+      "  Building wheel for fbprophet (setup.py): started\n",
+      "  Building wheel for fbprophet (setup.py): finished with status 'error'\n",
+      "  Running setup.py clean for fbprophet\n",
+      "  ERROR: Command errored out with exit status 1:\n",
+      "   command: /opt/conda/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-kzkf0jiy/fbprophet/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-kzkf0jiy/fbprophet/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' bdist_wheel -d /tmp/pip-wheel-my1e0lvf\n",
+      "       cwd: /tmp/pip-install-kzkf0jiy/fbprophet/\n",
+      "  Complete output (40 lines):\n",
+      "  running bdist_wheel\n",
+      "  running build\n",
+      "  running build_py\n",
+      "  creating build\n",
+      "  creating build/lib\n",
+      "  creating build/lib/fbprophet\n",
+      "  creating build/lib/fbprophet/stan_model\n",
+      "  Traceback (most recent call last):\n",
+      "    File \"<string>\", line 1, in <module>\n",
+      "    File \"/tmp/pip-install-kzkf0jiy/fbprophet/setup.py\", line 148, in <module>\n",
+      "      \"\"\"\n",
+      "    File \"/opt/conda/lib/python3.7/site-packages/setuptools/__init__.py\", line 144, in setup\n",
+      "      return distutils.core.setup(**attrs)\n",
+      "    File \"/opt/conda/lib/python3.7/distutils/core.py\", line 148, in setup\n",
+      "      dist.run_commands()\n",
+      "    File \"/opt/conda/lib/python3.7/distutils/dist.py\", line 966, in run_commands\n",
+      "      self.run_command(cmd)\n",
+      "    File \"/opt/conda/lib/python3.7/distutils/dist.py\", line 985, in run_command\n",
+      "      cmd_obj.run()\n",
+      "    File \"/opt/conda/lib/python3.7/site-packages/wheel/bdist_wheel.py\", line 223, in run\n",
+      "      self.run_command('build')\n",
+      "    File \"/opt/conda/lib/python3.7/distutils/cmd.py\", line 313, in run_command\n",
+      "      self.distribution.run_command(command)\n",
+      "    File \"/opt/conda/lib/python3.7/distutils/dist.py\", line 985, in run_command\n",
+      "      cmd_obj.run()\n",
+      "    File \"/opt/conda/lib/python3.7/distutils/command/build.py\", line 135, in run\n",
+      "      self.run_command(cmd_name)\n",
+      "    File \"/opt/conda/lib/python3.7/distutils/cmd.py\", line 313, in run_command\n",
+      "      self.distribution.run_com
```

Comment:
Why not use `kfserving` python SDK, that's very easy to deploy an inferenceservice. Thanks.
https://github.com/kubeflow/kfserving/tree/master/python/kfserving

Reply:
Updated this to use kfserving instead of manual yaml, thanks!